autoload_classmap.php

```diff
@@ -1,24 +0,0 @@
-<?php
-// Generated by ZF2's ./bin/classmap_generator.php
-return array(
-  'ZfcUserImpersonate\Module'                                               => __DIR__ . '/Module.php',
```

Comment:
I suggest moving the current `Module.php` to `src/ZfcUserImpersonate/Module.php` and adding a `/Module.php`   containing 
```php
<?php
require_once __DIR__ . '/src/ZfcUserImpersonate/Module.php';

```
That way we can omit the classmap in `composer.json` and the module becomes fully PSR-0/4 compliant.